### PR TITLE
feat(proposal page): add proposer filter to proposals page

### PR DESCRIPTION
**Pull request by stee1house**

### CODE REVIEW

File: astro_2.0/features/FeedProposals/FeedProposals.module.scss [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  padding: 10px;
+}
+
+.resultHint {
+  background: var(--color-white);
+  display: flex;
+  flex-direction: column;
+  padding: 8px;
+  width: 153px;
+  @media screen and (max-width: #{breakpoints.$screen-lg}px) {
```
Comment (AlexeyPonomarenko) [inline astro_2.0/features/FeedProposals/FeedProposals.module.scss]: I consider we have a mixin like `large-desktop` which can be used like
```
@include large-desktop {
  width: 100%;
}
```
Same for all medias above.
Not sure which usage is better. I consider you can discuss it with @VERZUOL1

File: astro_2.0/components/SearchInput/SearchInput.tsx [hunk 2]
```diff
@@ -213,29 +219,35 @@ export const SearchInput: FC<SearchInputProps> = ({
...
       <input
         ref={inputRef}
         tabIndex={0}
         value={value}
         onChange={handleChange}
-        className={cn(styles.input, 'body1')}
+        className={cn(
+          styles.input,
+          { [styles.withoutLoading]: !showLoader },
+          'body1'
```
Comment (AlexeyPonomarenko) [inline astro_2.0/components/SearchInput/SearchInput.tsx]: We have a typography `%body1` that can be used in your styles.input like
```
@extend %body1;
```

**Review by AlexeyPonomarenko — DISMISSED** [review]

File: astro_2.0/components/SearchInput/SearchInput.tsx (hunk 2 above)
Comment (stee1house) [reply astro_2.0/components/SearchInput/SearchInput.tsx]: It was there before me. But I will fix it. Thanks.

File: astro_2.0/features/FeedProposals/FeedProposals.module.scss (hunk 1 above)
Comment (stee1house) [reply astro_2.0/features/FeedProposals/FeedProposals.module.scss]: I will fix it. Thanks.

**Review by VERZUOL1 — APPROVED** [review]

**Pull request merged**